backend/src/main.py

```diff
@@ -148,7 +154,10 @@ def get_device(user_agent) -> str:
     )
     conn.get_account()
 else:
-    logger.warn('Variables necessary for OVH connection not set !')
+    conn = False
+    logger.warning('OVH credentials was not added')

```

Comment:
je préfère comme message 'OVH credential variables not found' s'il te plaît

Reply:
Pas de soucis